browser-extension/addRateLaterButton.css

```
@@ -20,3 +20,10 @@ img#tournesol-button-image {
   margin-right: 6px;
   vertical-align: middle;
 }
+
+iframe#x-tournesol-iframe {
+  display: none;
+
+  width: 100%;
+  height: 490px;
```

To make the iframe a bit more esthetically pleasing, by removing the default border:
```suggestion
  height: 490px;
  border: none;
  margin: 0;
```

---

Fixed by https://github.com/tournesol-app/tournesol/pull/492/commits/b267e4d736e984a30339a947ee32e1cf353ab468